### PR TITLE
[MPMD-404] Bump PMD to 7.7.0

### PR DESCRIPTION
**Pull request by pzygielo**
- Closes #168

### CODE REVIEW

**Review by slachiewicz — APPROVED** [review]

**Comment by michael-o** [comment]
Like I said, before I release the plugin again there will be a hundred new releases in between.

**Comment by michael-o** [comment]
7.7.0 should be released next week. Will wait for it. @adangel

File: pom.xml [hunk 1]
```diff
@@ -83,7 +83,7 @@ under the License.
   <properties>
     <javaVersion>8</javaVersion>
     <mavenVersion>3.6.3</mavenVersion>
-    <pmdVersion>7.3.0</pmdVersion>
+    <pmdVersion>7.6.0</pmdVersion>
```
Comment (adangel) [inline pom.xml:86]:
```suggestion
    <pmdVersion>7.7.0</pmdVersion>
```

File: src/site/apt/examples/upgrading-PMD-at-runtime.apt.vm [hunk 2]
```diff
@@ -88,6 +88,8 @@ Upgrading PMD at Runtime
 *--------------------------------------------------------------------------------*-----------------------------------------------------*
 | <<maven-pmd-plugin>>                                                           | <<PMD>>                                             |
 *--------------------------------------------------------------------------------*-----------------------------------------------------*
+| {{{https://maven.apache.org/plugins-archives/maven-pmd-plugin-3.26.0/}3.26.0}} | {{{https://docs.pmd-code.org/pmd-doc-7.6.0/}7.6.0}} |
```
Comment (adangel) [inline src/site/apt/examples/upgrading-PMD-at-runtime.apt.vm:91]:
```suggestion
| {{{https://maven.apache.org/plugins-archives/maven-pmd-plugin-3.26.0/}3.26.0}} | {{{https://docs.pmd-code.org/pmd-doc-7.7.0/}7.7.0}} |
```

**Review by adangel — COMMENTED** [review]
PMD 7.7.0 is there...

**Comment by michael-o** [comment]
> PMD 7.7.0 is there...

Magic, thanks. Will merge as soon as CI finishes.

**Comment by jira-importer** [comment]
Resolve #371

**Comment by jira-importer** [comment]
Resolve #371

**Pull request closed without merging**